internal/sherlock.go

```diff
@@ -250,6 +250,10 @@ func SplitQuery(query string) (string, string, error) {
 	return set[0], set[1], nil
 }
 
+func NameValidation(name string) bool {
+	return !strings.Contains(name, "@")
```

Comment:
here we can use the `querySplitPoint` which is declared in the `internal.sherlock.go` file.

Reply:
I see, My bad. I'll update soon